src/material/core/tokens/_token-utils.scss

```diff
@@ -3,11 +3,11 @@
 @use 'sass:string';
 @use '../style/elevation';
 @use '../style/sass-utils';
-@use './m3-tokens';
+@use 'm3-system';
```

Comment:
Should this be a relative path?

Reply:
Yeah thanks - looks like Webstorm changes this in a rename